src/backend/oneapi/CMakeLists.txt

```diff
@@ -264,15 +266,20 @@ add_library(ArrayFire::afoneapi ALIAS afoneapi)
 
 arrayfire_set_default_cxx_flags(afoneapi)
 
+find_package(MKL CONFIG REQUIRED)
+message(STATUS "${MKL_IMPORTED_TARGETS}")
```

Comment:
Master now has the changes to correctly include the MKLConfig file from the oneapi install. Please remove